bundles/core/src/test/resources/aemform/test-content.json

```diff
@@ -6,7 +6,7 @@
             "jcr:title"         : "AEM Form Test",
             "sling:resourceType": "core/wcm/components/page/v2/page",
             "jcr:description"   : "AEM Form description",
-            "sling:configRef"   : "/conf/core-components-examples/content",
+            "sling:configRef"   : "/conf/fileinput/content",
```

Comment:
is this change really required?
cc @rismehta

Reply:
no